examples/js/components/Map.jsx

```diff
@@ -10,6 +12,13 @@ class Map extends Component {
            title="Map + Table"
            subtitle="Tab Layout"
          />
+         <DefaultMap
```

Comment:
Why? This was only added because it did not display on the left (default position) due to the left panel overlaying over it. If we add layer control, that would go on the right with zoom control on the left (default).